ext/standard/tests/file/005_variation-win32.phpt

```diff
@@ -5,6 +5,7 @@ Test fileatime(), filemtime(), filectime() & touch() functions : usage variation
 if (substr(PHP_OS, 0, 3) != 'WIN') {
     die('skip Windows-only test');
 }
+if (getenv("SKIP_SLOW_TESTS")) die("skip slow test");
```

Comment:
we should print microseconds as well and sleep for like 10 ms instead of 1 s

Reply:
filemtime() and friends return Unix timestamps, so there are no fractions of a second.

Reply:
`ext4` and other modern systems supports better time than second resolution, `filemtime` should be probably fixed

src:
https://stackoverflow.com/a/19100503/5153116
https://stackoverflow.com/questions/24804618/get-file-mtime-with-millisecond-resolution-from-java
https://stackoverflow.com/questions/51260051/file-last-modified-time-with-milliseconds-precision

Reply:
"Fixing" filemtime() to return float, would be a BC break, though. We would need to introduce a new set of functions or optional parameters to avoid that.